test/util/test_app.go

```diff
@@ -63,7 +64,17 @@ func (ao EmptyAppOptions) Get(_ string) interface{} {
 // of the app from first genesis account. A no-op logger is set in app.
 func SetupTestAppWithGenesisValSet(cparams *tmproto.ConsensusParams, genAccounts ...string) (*app.App, keyring.Keyring) {
 	testApp, valSet, kr := NewTestAppWithGenesisSet(cparams, genAccounts...)
+	initialiseTestApp(testApp, valSet, cparams)
+	return testApp, kr
+}
+
+func SetupTestAppWithGenesisValSetAndMaxSquareSize(cparams *tmproto.ConsensusParams, maxSquareSize int, genAccounts ...string) (*app.App, keyring.Keyring) {
+	testApp, valSet, kr := NewTestAppWithGenesisSetAndMaxSquareSize(cparams, maxSquareSize, genAccounts...)
+	initialiseTestApp(testApp, valSet, cparams)
```

Comment:
> initiali~~s~~eTestApp

:zany_face: :innocent: 

![zorro z](https://github.com/user-attachments/assets/182989e7-6234-4b54-b8ec-ce94104fe65f)

Reply:
British vs American 😆 . I guess I need to set my IDE to American English because I always need to fix the British differences

Reply:
yeah lol no changes needed!